mesonbuild/interpreter/primitives/integer.py

```diff
@@ -65,10 +65,13 @@ def is_even_method(self, args: T.List[TYPE_var], kwargs: TYPE_kwargs) -> bool:
     def is_odd_method(self, args: T.List[TYPE_var], kwargs: TYPE_kwargs) -> bool:
         return self.held_object % 2 != 0
 
-    @noKwargs
+    @typed_kwargs(
+        'to_string',
+        KwargInfo('fill', int, default=0, since='1.3.0')
+    )
     @noPosargs
     def to_string_method(self, args: T.List[TYPE_var], kwargs: TYPE_kwargs) -> str:
```

Comment:
It's because kwargs does not have a proper type, you have to create TypeDict, see other methods that have kwargs.

Reply:
1. `T.Dict[str, T.Any]` is not a TypedDict
2. Passing "dict of `str: T.Any`" is basically telling the type checker to "shut up", since an Any type is saying "allow anything here": https://mypy.readthedocs.io/en/stable/dynamic_typing.html#dynamic-typing